README.md

```diff
@@ -0,0 +1,4 @@
+# Hypershift Audit Logs Forwarder
+
+Hypershift logging forwarder is an operator that lives in each hosted control plane (HCP) namespace in management clusters.
+The main goal of this operator is to forward logs from HCP and hosted cluster to a definable destination. 
```

Comment:
```suggestion
The Hypershift Logging Operator combines data-plane and control-plane configuration on the control plane, so that the Cluster Logging Operator on the management cluster can forward API audit logs for both personas. 
```